### PR TITLE
[TT-418] automation e2e docker migration

### PR DESCRIPTION
**Pull request by tateexon**

### CODE REVIEW

**Review by skudasov — APPROVED** [review]

**Review by anirudhwarrier — CHANGES_REQUESTED** [review]
Going through the CI runs from these changes, it seems none of the tests themselves are running.
https://github.com/smartcontractkit/chainlink/actions/runs/6090501039/job/16542178035?pr=10432#step:5:541
```
Running Smoke Test on Simulated Geth
Chainlink Image ***.dkr.ecr.***.amazonaws.com/chainlink
Chainlink Version 2980a7c3227cedc324726bee9c0f26297578ade2
testing: warning: no tests to run
ok  	command-line-arguments	0.140s [no tests to run]
```

Is this expected?

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -111,6 +111,7 @@ jobs:
           echo "\`${GITHUB_SHA}\`" >>$GITHUB_STEP_SUMMARY
 
   build-test-image:
+    if: startsWith(github.ref, 'refs/tags/') || github.event_name == 'schedule' || (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/'))
```
Comment (kalverra) [inline .github/workflows/integration-tests.yml:114]: Are we able to trigger this on certain labels as well? It's a common workflow to open a PR when making config changes to long-running tests, and pull the image from that build.

Comment (tateexon) [reply .github/workflows/integration-tests.yml:114]: What label would you like to use?

Comment (tateexon) [reply .github/workflows/integration-tests.yml]: I added the ability to trigger this with the label 'build-test-image'

**Review by anirudhwarrier — APPROVED** [review]
Looks good now. Created a [separate task](https://smartcontract-it.atlassian.net/browse/TT-572) to check why some keeper smoke tests are running sequentially while others are parallel.

**Pull request merged**